falco.yaml

```diff
@@ -460,87 +460,108 @@ metadata_download:
+# (overriding rules), else remove unwanted syscalls from the Falco rules.
+#
+# Passing `-o "log_level=debug" -o "log_stderr=true" --dry-run` to Falco's
+# cmd args will print the final set of syscalls to STDOUT.
 #
 # --- [Suggestions]
 #
-# Here are a few recommendations that may help you.
-# Setting `base_syscalls.repair: true` automates these recommendations for you.
+# NOTE: setting `base_syscalls.repair: true` automates the following suggestions for you.
 #
-# Consider to at minimum add the following syscalls regardless of the syscalls used in the rules.
+# These suggestions are subject to change as Falco and its state engine evolve.
 #
-# [clone, clone3, fork, vfork, execve, execveat, close]
+# For execve* events:
+# Some Falco fields for an execve* syscall are retrieved from the associated
+# `clone`, `clone3`, `fork`, `vfork` syscalls when spawning a new process.
+# The `close` syscall is used to purge file descriptors from Falco's internal
+# thread / process cache table and is necessary for rules relating to file
+# descriptors (e.g. open, openat, openat2, socket, connect, accept, accept4 ... and many more)
 #
-# This is because some Falco fields for an execve* system call are retrieved
-# from the associated `clone`, `clone3`, `fork`, `vfork` syscall when spawning a
-# new process. The `close` system call is used to purge file descriptors from Falco's
-# internal thread / process cache table and should always be added when you have
-# rules around file descriptors.
-# (e.g. open, openat, openat2, socket, connect, accept, accept4 ... and many more)
+# Consider enabling the following syscalls in `base_syscalls.custom_set` for process rules:
+# [clone, clone3, fork, vfork, execve, execveat, close]
 #
-# When network syscalls are used in rules we recommend to at minimum set
+# For networking related events:
+# While you can log `connect` or `accept*` syscalls without the socket syscall,
+# the log will not contain the ip tuples.
+# Additionally, for `listen` and `accept*` syscalls, the `bind` syscall is also necessary.
 #
+# We recommend the following as the minimum set for networking-related rules:
 # [clone, clone3, fork, vfork, execve, execveat, close, socket, bind, getsockopt]
 #
-# It turns out that while you can log `connect` or `accept*` syscalls without the
-# socket system call, the log however would not contain the ip tuples.
-# For `listen` and `accept*` system calls you also need the `bind` system call.
-#
-# Lastly, if you care about the correct `uid`, `gid` or `sid`, `pgid` of a process when the
-# running process opens a file or makes a network connection, consider adding the following syscalls:
+# Lastly, for tracking the correct `uid`, `gid` or `sid`, `pgid` of a process when the
+# running process opens a file or makes a network connection, consider the following syscall set:
+# [setresuid, setsid, setuid, setgid, setpgid, setresgid, setsid, capset, chdir, chroot, fchdir]
```

Comment:
@Happy-Dude may I kindly ask to make another super small modification here?  I think before in was not as a list, would recommend to not have it as a list here in brackets  and instead in the following form:
` ... setresuid, setsid, setuid, setgid, setpgid, setresgid, setsid, capset, chdir, chroot, fchdir ... `

change language to `consider adding the following syscalls to the above recommended syscall sets:`

Reply:
Done in latest push + rebase :)